heroic-test-it/src/main/java/com/spotify/heroic/test/AbstractSuggestBackendIT.java

```diff
@@ -192,6 +207,75 @@ public void keySuggestTest() throws Exception {
         assertEquals(ImmutableSet.of(s1.getKey(), s2.getKey()), result);
     }
 
+    @Test
+    public void tagValueSuggestNoIdx() throws Exception {
+        final TagValueSuggest.Request request =
+            new TagValueSuggest.Request(TrueFilter.get(), range, OptionalLimit.empty(),
+                Optional.of("role"));
+
+        final TagValueSuggest result = backend.tagValueSuggest(request).get();
+
+        assertEquals(Collections.emptyList(), result.getValues());
```

Comment:
The fixture (and in the following tests) are duplicated. We should figure out a way to avoid that.